broker-daemon/bin/sparkswapd.js

```diff
@@ -5,97 +5,104 @@
+  .option('--rpc.pass [rpc-pass]', 'Broker rpc password', program.String, config.rpc.pass)
+  .option('--rpc.pub-key [rpc-pub-key]', 'Location of the public key for the broker\'s rpc', validations.isFormattedPath, config.rpc.pubKey)
+  .option('--rpc.priv-key [rpc-priv-key]', 'Location of private key for the broker\'s rpc', validations.isFormattedPath, config.rpc.privKey)
 
+// For each currency, we will add expected options for an engine
 for (let currency of currencies) {
   let lowerSymbol = currency.symbol.toLowerCase()
-  let ENGINE_TYPE = process.env[`${currency.symbol}_ENGINE_TYPE`]
-  let LND_TLS_CERT = process.env[`${currency.symbol}_LND_TLS_CERT`]
-  let LND_MACAROON = process.env[`${currency.symbol}_LND_MACAROON`]
-  let LND_RPC_HOST = process.env[`${currency.symbol}_LND_RPC_HOST`]
+
   program
-    .option(`--${lowerSymbol}-engine-type <type>`, `The type of underlying Payment Channel Network node for ${currency.name}`, [ 'LND' ], ENGINE_TYPE)
+    .option(`--${lowerSymbol}.engine-type [engine-type]`, `The type of underlying Payment Channel Network node for ${currency.name}`, isSupportedEngineType, currency.engineType)
+    .option(`--${lowerSymbol}.rpc-host [rpc-host]`, `Location of a Payment Channel Network node's RPC server to use for ${currency.name}.`, validations.isHost, currency.rpcHost)
+    .option(`--${lowerSymbol}.tls-cert [tls-cert]`, `Location of the TLS certificate for the Payment Channel Network node to use when communicating with ${currency.name}.`, validations.isFormattedPath, currency.tlsCert)
+
   // LND Specific commands
-  // These will be validated based off of the engine type
   program
-    .option(`--${lowerSymbol}-lnd-rpc <server>`, `Location of the LND RPC server to use for ${currency.name}.`, validations.isHost, LND_RPC_HOST)
-    .option(`--${lowerSymbol}-lnd-tls <path>`, `Location of the certificate to use when communicating with ${currency.name} LND.`, validations.isFormattedPath, LND_TLS_CERT)
-    .option(`--${lowerSymbol}-lnd-macaroon <path>`, `Location of the macaroon to use when communicating with ${currency.name} LND.`, validations.isFormattedPath, LND_MACAROON)
+    .option(`--${lowerSymbol}.lnd-macaroon [lnd-macaroon]`, `Location of the LND macaroon to use when communicating with ${currency.name} LND. (Only for LND engine type)`, validations.isFormattedPath, currency.lndMacaroon)
 }
 
 program
   .action((args, opts) => {
     const {
-      rpcAddress,
       dataDir,
-      markets,
       interchainRouterAddress,
-      relayerHost: relayerRpcHost,
-      relayerCertPath,
-      idPrivkeyPath: privIdKeyPath,
-      idPubkeyPath: pubIdKeyPath,
+      idPubKey: pubIdKeyPath,
+      idPrivKey: privIdKeyPath,
+      markets,
       disableAuth,
-      rpcPrivkeyPath: privRpcKeyPath,
-      rpcPubkeyPath: pubRpcKeyPath,
+      relayerRpcHost,
+      relayerCertPath,
+      rpcAddress,
       rpcUser,
-      rpcPass
+      rpcPass,
+      rpcPubKey: pubRpcKeyPath,
```

Comment:
These are tough, because you probably want to keep the `path` suffix for the BrokerDaemon options, but at the very least these (and the id ones) would make more sense to me if the options the BrokerDaemon took were `rpcPubKeyPath` and `idPrivKeyPath` rather than the current setup.